scalalib/src/mill/scalalib/Lib.scala

```diff
@@ -27,47 +26,51 @@ object Lib{
       .toIO
   }
 
-  /**
-    * Resolve dependencies using Coursier.
-    *
-    * We do not bother breaking this out into the separate ScalaWorker classpath,
-    * because Coursier is already bundled with mill/Ammonite to support the
-    * `import $ivy` syntax.
-    */
-  def resolveDependencies(repositories: Seq[Repository],
-                          scalaVersion: String,
-                          deps: TraversableOnce[Dep],
-                          platformSuffix: String = "",
-                          sources: Boolean = false): Result[Agg[PathRef]] = {
-
-    val flattened = deps.map{
+  def depToDependency(dep: Dep, scalaVersion: String, platformSuffix: String = ""): Dependency =
+    dep match {
       case Dep.Java(dep, cross) =>
         dep.copy(
           module = dep.module.copy(
             name =
               dep.module.name +
-              (if (!cross) "" else platformSuffix)
+                (if (!cross) "" else platformSuffix)
```

Comment:
Could you format these the old way?

Reply:
Sure, missed that. That's IntelliJ. Do you have a set of IntelliJ defaults for this or just have auto formatting turned off?